utils/eventtracking.js

```diff
@@ -53,15 +53,18 @@ const track = async (eventType, eventProperties, options) => {
             const id = getMixpanelID(shellSettings);
             await Promise.all([
                 mixpanel.alias(accountID, id),
-                mixpanel.people.set(id, {account_id: accountID})
+                mixpanel.people.set(id, { account_id: accountID })
             ]);
         }
 
+        const user_country = await getUserCountry();
```

Comment:
when `getUserCountry` throws an error and `console.log()` it, won't the return value be `undefined` here? And in that case, shouldn't we be gracefully handling it here or in the function itself?

Reply:
Now we will just log the message. How do you want to handle it? If this call was unsuccessful - we should not return any errors. The user does not care if it was successful. Same reason for `.log()` vs `.error()`. Also, it's ok to send `undefined`, because this is what happened. The country is undefined.

Reply:
I see, if undefined is expected then its fine